docker/docker-compose.yml

```diff
@@ -218,8 +218,8 @@ services:
       dockerfile: docker/sbtc/Dockerfile
       target: blocklist-client
     environment:
-      BLOCKLIST_CLIENT_RISK_ANALYSIS__API_URL: ${BLOCKLIST_CLIENT_RISK_ANALYSIS__API_URL:-https://public.chainalysis.com}
-      BLOCKLIST_CLIENT_RISK_ANALYSIS__API_KEY: ${BLOCKLIST_CLIENT_RISK_ANALYSIS__API_KEY:-}
+      BLOCKLIST_CLIENT__RISK_ANALYSIS__API_URL: ${BLOCKLIST_CLIENT__RISK_ANALYSIS__API_URL:-https://public.chainalysis.com}
```

Comment:
Ahh, travelling now but I'm going to revert this. I see that the `Environment` needs `.with_prefix_separator("_")` like we do in the signer. Must have defaulted to `_` in the old version.

Reply:
Still to do?

Reply:
Updated in [c15b3e8](https://github.com/stacks-network/sbtc/pull/1271/commits/c15b3e821399fcba3e22752a8a8626de1b7e4bc6)